_includes/footer.html

```diff
@@ -0,0 +1,40 @@
+<footer class="site-footer h-card">
+  <data class="u-url" href="{{ "/" | relative_url }}"></data>
+
+  <div class="wrapper">
+
+    <div class="footer-col-wrapper">
+
+      <div class="footer-col footer-col-1">
+        <h6>Contact Us</h6>
+        <ul class="contact-list">
+          <!-- <li class="p-name">
+            {%- if site.author -%}
+              {{ site.author | escape }}
+            {%- else -%}
+              {{ site.title | escape }}
+            {%- endif -%}
+            </li> -->
+            {%- if site.email -%}
+            <li>
+              <svg class="svg-icon">
+                <use xlink:href="{{relativeBase}}architecture/images/mail.svg#mail"></use>
+              </svg>
+              <a class="u-email" href="mailto:{{ site.email }}">{{ site.email }}</a></li>
+            {%- endif -%}
+        </ul>
+        {%- include social.html -%}
```

Comment:
>jekyll 3.6.2 | Error:  Could not locate the included file 'social.html'

@nickbrandt are you able to build this locally? I realize this is probably directly from the underlying template...

Reply:
@devinreams Hmmm...yea I have no issues building this locally.